--- missing.py	2020-12-06 13:54:22.000000000 +0300
+++ missing.py	2020-12-06 13:54:00.000000000 +0300
@@ -157,6 +157,8 @@
         return _isna_ndarraylike(obj, inf_as_na=inf_as_na)
     elif isinstance(obj, ABCDataFrame):
         return obj.isna()
+    elif callable(getattr(obj, "isna", None)):
+        return obj.isna()
     elif isinstance(obj, list):
         return _isna_ndarraylike(np.asarray(obj, dtype=object), inf_as_na=inf_as_na)
     elif hasattr(obj, "__array__"):
